doc/unison-manual.tex

```diff
@@ -1995,9 +1995,29 @@
 monitoring), interrupting with ``Ctrl-C'' or with signal \verb|SIGINT| or
 \verb|SIGTERM| works the same way as described above and will additionally stop
 the continuous process. To stop only the continuous process and let the last
-synchronization complete normally, send signal \verb|SIGUSR2| or \verb|SIGQUIT|
-(the latter can also be generated from keyboard by pressing
-``Ctrl-\textbackslash'' or ``Ctrl-4''), instead.
+synchronization complete normally, press ``Ctrl-D'' or send signal
+\verb|SIGUSR2| or \verb|SIGQUIT| (the latter can also be generated from
+keyboard by pressing ``Ctrl-\textbackslash'' or ``Ctrl-4''), instead.
+``Ctrl-D'' is expected to send an EOF if the sandard input is a terminal.
+Closing the input and receiving an EOF, a \verb|^C| (0x03), or a \verb|^D|
+(0x04) from the input terminal or a redirected standard input have the same
+effect. If the standard input is redirected from a regular file, not open or
+not open for reading already at Unison startup then it is ignored and only
+signals can be used to send the stop request.
+
+\begin{quote}
+  {\em Tips:} For continuous synchronization the input should not be
+  redirected from \verb|/dev/null| (as that would stop the process after
+  the first synchronization). This is what some implementations of \verb|nohup|
+  may do if run from a terminal without redirecting input. Input should
+  neither be redirected from any source providing a lot of input (such as
+  a device or a pipe that produces large quantities of data). In such cases,
+  Unison would keep reading all the input looking for the stop condition.
+
+  Pressing ``Ctrl-D'' will work properly if pressed at start of a line. You
+  may have to press ``Enter'' first to start a new line if there is a chance
+  that the input buffer is not empty.
```

Comment:
I would be inclined to just talk about EOF, and not veer into unix tutorial, as there's nothing special about unison.

I wonder if this is really needed; with a signal, esp double SIGINT, it seems like a special case.    It also sort of opens the door to interactive commands on stdin for all sorts of things, and I don't see a reason to open that box in general.

Reply:
> I would be inclined to just talk about EOF, and not veer into unix tutorial, as there's nothing special about unison.

Yes, I will reduce the verbosity in the manual.

> I wonder if this is really needed; with a signal, esp double SIGINT, it seems like a special case. It also sort of opens the door to interactive commands on stdin for all sorts of things, and I don't see a reason to open that box in general.

No signals in Windows... there is a need for a non-signal based method if this is expected to work in Windows.